.travis.yml

```diff
@@ -1,10 +1,17 @@
 language: ruby
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.1
-  - 2.2.3
-  - 2.3.0
+- 2.0.0
+- 2.1.1
+- 2.2.3
+- 2.3.0
+- 2.4.2
```

Comment:
I think this array should feature latest 2.3, 2.4 and 2.5 since 2.0, 2.1 and 2.2 have reached EOL. Then, what about setting the travis condition to 2.5.1 ?

Reply:
Sounds good! Bumped the versions along with `.ruby-version` d2ab862

(Any other places we need to update? Didn't find anywhere we mention the minimum Ruby version)